### DISCUSSION TITLE
Is there any standalone App based on milkdown?

### DISCUSSION THREAD

**Discussion by yingbo** [Milkdown/milkdown#873] [General]
I'm searching Typora alternative and found this one. Just curious is there an App already based on the milkdown lib/framework?

**Comment by Saul-Mirone** [comment]
> I'm searching Typora alternative and found this one. Just curious is there an App already based on the milkdown lib/framework?

You can view the list here: https://github.com/Saul-Mirone/awesome-milkdown
